src/main/webapp/src/app/admin-page/admin-page.component.ts

```diff
@@ -159,6 +165,63 @@ export class AdminPageComponent implements OnInit {
     });
   }
 
+  /**
+   * Posts a plant to a garden.
+   *
+   * Performs POST: /garden/{{gardenProfile.id}}/plant-list, with
+   * body plant.
+   *
+   * @param plant the plant to post.
+   * @return the http response.
+   */
+  postPlantToGarden(plant: Plant): Observable<HttpResponse<string>> {
+    console.log(JSON.stringify(plant));
+    return this.httpClient.post<string>(
+      '/garden/' + this.gardenProfile.id + '/plant-list',
+      plant,
+      {
+        observe: 'response',
+        responseType: 'json',
+      }
+    );
+  }
+
+  /**
+   * Deletes a plant from a garden.
+   *
+   * Performs GET: /garden/{{gardenProfile.id}}/plant-list/{id}
+   *
+   * @param id the plant id to delete.
+   * @return the http response.
+   */
+  deletePlant(id: string): Observable<HttpResponse<string>> {
+    return this.httpClient.delete<string>(
+      '/garden/' + this.gardenProfile.id + '/plant-list/' + id,
+      {
+        observe: 'response',
```

Comment:
What is the default `observe` value for DELETE/POST? Can we omit the extra opts or are they necessary

Reply:
The default `observe` value is the response body, whereas I use the entire response. Though I do think the changes you suggest here would be easier to implement than the changes you suggest in #41 with `GET` requests.